inc/thorvg.h

```diff
@@ -1190,6 +1203,19 @@ class TVG_API Shape final : public Paint
      */
     float strokeMiterlimit() const noexcept;
 
+    /**
+     * @brief Gets the trim of the stroke along the defined path segment.
+     *
+     * @param[out] begin The starting point of the segment to display along the path.
+     * @param[out] end Specifies the end of the segment to display along the path.
+     * @param[out] simultaneous Determines how to trim multiple paths within a single shape.
+     *
+     * @retval Result::Success when succeed, Result::InsufficientCondition otherwise.
+     *
+     * @note Experimental API
+     */
+    Result strokeTrim(float* begin, float* end, bool* simultaneous) const noexcept;
```

Comment:
Return type is useless practically. let's see this suggestion. we can make apis simpler.
```
// return true when simultaneous, false otherwise.
bool strokeTrim(float* begin, float* end) const noexcept;
```